cockatrice/src/client/network/replay_timeline_widget.cpp

```diff
@@ -147,17 +147,24 @@ void ReplayTimelineWidget::replayTimerTimeout()
 {
     currentTime += 200;
 
-    processNewEvents();
+    processNewEvents(NORMAL_PLAYBACK);
 
     if (!(currentTime % 1000))
         update();
 }
 
 /// Processes all unprocessed events up to the current time.
-void ReplayTimelineWidget::processNewEvents()
+void ReplayTimelineWidget::processNewEvents(PlaybackMode playbackMode)
 {
     while ((currentEvent < replayTimeline.size()) && (replayTimeline[currentEvent] < currentTime)) {
-        emit processNextEvent();
+        Player::EventProcessingOptions options;
+
+        // backwards skip => always skip reveal windows
+        // forwards skip => skip reveal windows that don't happen within 10 seconds of the target
+        if (playbackMode == BACKWARD_SKIP || currentTime - replayTimeline[currentEvent] > 10000)
```

Comment:
I'd like this 10 000 to be made into a constant with unit

Reply:
I already have a PR in the works to move all the magic numbers in ReplayTimeWidget into constants, but it builds upon this PR and so needs it to be merged in first

Reply:
so you'd like me to merge this in before that?

Reply:
yeah